index.bs

```diff
@@ -2594,6 +2611,10 @@ party.
 : Client processing
 :: None, except default forwarding of client argument to authenticator argument.
 
+: Client data
+:: Returns a JSON object that encodes the location information in the authenticator data as a Coordinates value,
+    as defined by [The W3C Geolocation API Specification](https://dev.w3.org/geo/api/spec-source.html#coordinates_interface).
+
```

Comment:
ClientData (which must be hashed and fed into the authenticator for signature) cannot depend on AuthenticatorData (which comes out of the authenticator after the signature has been computed). Should be "none".